### PR TITLE
add basic layout in slim, header, footer partial and hocus pocus css

### PR DESCRIPTION
**Pull request by bpietraga**
connected to #30

### CODE REVIEW

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -1,10 +1,21 @@
 doctype html
 html
   head
+<<<<<<< c47154a94e73a4bb3a4f4da8b4ac4ad8973e1712
```
Comment (Snieezy) [inline app/views/layouts/application.html.slim]: i think those are leftovers from conficts? you should remove them

File: db/schema.rb [hunk 2]
```diff
@@ -1,3 +1,15 @@
+# This file is auto-generated from the current state of the database. Instead
```
Comment (Snieezy) [inline db/schema.rb]: should we keep those comments?

**Comment by hav3loc** [comment]
I think we shouldn't try to merge more things into this branch, let it be merged and then go with PR to master.

File: db/schema.rb (hunk 2 above)
Comment (hav3loc) [inline db/schema.rb]: I think they don't matter much, but we can delete them.

File: app/assets/stylesheets/_defaults.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* imports hocus-pocus colors */
```
Comment (smt116) [inline app/assets/stylesheets/_defaults.scss]: this comment is useless

File: app/assets/stylesheets/_defaults.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+/* typography */
+$font-leading: 1.4;
+$font-size-base: 15;
+$font-family-base: "Open Sans", sans-serif;
+
+/* responsive design */
+$rwd-type: mobile-first;
+$rwd-map: (tablet: 768px 1200px, desktop: 1201px);
+
+/* whole hocus-pocus */
```
Comment (smt116) [inline app/assets/stylesheets/_defaults.scss]: same here

File: app/views/layouts/_footer.html.slim [hunk 5]
```diff
@@ -0,0 +1,8 @@
+footer.sticky-footer
+  .container
+    .grid.grid-pair
+      .grid-item class="1/2"
+        group group III
+      .grid-item class="1/2"
+        group Internship
+
```
Comment (smt116) [inline app/views/layouts/_footer.html.slim]: please, configure your editor

File: app/views/layouts/_header.html.slim [hunk 6]
```diff
@@ -0,0 +1,19 @@
+.navbar
+  .container
+    .grid.grid-pair
+      .grid-item class="1/4"
+        a.navbar-link href="#"  Board&Roll#Logogoeshere
```
Comment (smt116) [inline app/views/layouts/_header.html.slim]: "#logogoeshere"?

File: app/views/layouts/_header.html.slim [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+      .grid-item class="3/4"
+        ul.navbar-menu
+          li = link_to '#', data: {toggle: 'tab'} do
+               span
+                 = "Events"
```
Comment (smt116) [inline app/views/layouts/_header.html.slim]: indentation? you should probably break a line before `= link_to ...`

File: app/views/layouts/application.html.slim [hunk 8]
```diff
@@ -1,10 +1,12 @@
 doctype html
 html
   head
-    title
-      | BoardGames
+    meta content=("text/html; charset=UTF-8") http-equiv="Content-Type" /
```
Comment (smt116) [inline app/views/layouts/application.html.slim]: is a slash needed?

File: db/schema.rb [hunk 9]
```diff
@@ -19,5 +18,4 @@
     t.index ["email"], name: "index_users_on_email", unique: true
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
-
 end
```
Comment (smt116) [inline db/schema.rb]: this file should be generated automatically. Do not change it on your own.

**Comment by smt116** [comment]
8ab171e commit message is too long.

**Pull request closed without merging**